### ISSUE TITLE
Merge all current registration work onto QA

### ISSUE THREAD

**Issue by rsgonzal** [Iridescent-CM/technovation-app#3055]
- New sign up flow
- Configuration for tailwind
- Vue formulate library

**Comment by viviancan** [comment]
Current Status 10/7/21

I had a build failure on heroku where the tailwindcss module wasnt found!

```
remote:            Module build failed (from ./node_modules/postcss-loader/src/index.js):        
remote:            Error: Cannot find module 'tailwindcss'        
```

I did some research and saw a few solutions to move the needed dependencies from devDependencies into dependencies in the package.json file. So I did that and pushed back to my branch. Just waiting for those tests to finish! Going to verify with shaun!

**Comment by viviancan** [comment]
I am currently working through this. It has been quite the chase, as there are no error logs to start with.

Here are some resources I have been using
- https://edgeguides.rubyonrails.org/webpacker.html
- https://stackoverflow.com/questions/65858869/webpacker-cant-find-application-css-in-app-public-packs-manifest-json
- https://rubyyagi.com/tailwind-css-on-rails-6-intro/
-

**Comment by viviancan** [comment]
I made a few updates and tailwindcss is now live at `/new-registration`. After tailwind was properly compiling, I noticed some of the custom styles were still missing. I tried a few things and finally found [this reddit post](https://www.reddit.com/r/tailwindcss/comments/l0z7in/tailwind_styles_missing_when_deployed_to_heroku/
) about the "purge" functionality. Once removed, the custom tailwind css displayed properly. I am adding this to the list of cleanup tasks, for something to possible revisit! 

https://tailwindcss.com/docs/optimizing-for-production

**Comment by viviancan** [comment]
This issue has once again popped up 👻  The Webpacker Ghost 😱 

After pushing #3035 (which also included #3071) which is the student my profile tab UI rebranding, I tested QA and noticed all the styles are jumbled. After further investigation, it appears that the tailwind styles are not being loaded properly through webpacker. This is a little different that what I encountered previously (^^ the purge issue) in that the student my-profile are just regular rails views compared to a separate vue.js app (like new registration). After speaking with Shaun, we have found a potential fix, by adding the `stylesheet pack tag` to the application_rebrand layout. 

Here are some additional sources

- https://stackoverflow.com/questions/60471982/when-to-use-stylesheet-pack-tag-instead-stylesheet-link-tag-with-rails-6
- https://github.com/rails/webpacker/issues/2343
- https://github.com/rails/webpacker/issues/1575
- https://github.com/rails/webpacker/tree/5-x-stable#usage
